toolkit/tools/specreader/specreader.go

```diff
@@ -254,7 +255,7 @@ func readSpecWorker(requests <-chan string, results chan<- *parseResult, cancel
 
 	defer wg.Done()
 
-	defines := rpm.DefaultDefines()
+	defines := rpm.DefaultDefines(*runCheck)
```

Comment:
I know it's not pretty, but please pass this as an argument to the function instead of using the global. That's the convention we've been following (at least in this file) and introducing something new creates a bigger mess.

Reply:
Thanks- redid the PR to pass through the runCheck variable as function args all around.